src/main/java/com/itschool/eventmanagment/exceptions/EventNotFoundException.java

```diff
@@ -3,7 +3,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(value= HttpStatus.NOT_FOUND, reason="No such Order")  // 404
+@ResponseStatus(value= HttpStatus.NOT_FOUND, reason="No such id")  // 404
```

Comment:
Please consider adding a static import for `HttpsStatus` to simplify things and remove the comment. If someone doesn't know what `NOT_FOUND` stands for, there is documentation for this.

Also, handling custom exceptions should fall under the responsibility of an `ExceptionHandlerAdvice`. You have an example [here](https://github.com/AlexandruIliescu/IT-School-GR4-Spring-Boot/blob/main/src/main/java/com/itschool/project/exceptions/ExceptionHandlerAdvice.java).

Reply:
@AlexandruIliescu Thanks. Since this PR has deviated from its initial purpose and new changes have been requested each time despite the back-and-forth, I will create a new PR on the dev branch to be resolved against the main branch, in order to address these changes in an organized manner.